test/test_write_concern.py

```diff
@@ -0,0 +1,32 @@
+import collections
+import unittest
+
+from pymongo.write_concern import WriteConcern
+
+
+class TestWriteConcern(unittest.TestCase):
+
+    def test_equality(self):
+        concern = WriteConcern(j=True, wtimeout=3000)
+        self.assertEqual(concern, WriteConcern(j=True, wtimeout=3000))
+        self.assertNotEqual(concern, WriteConcern())
+
+    def test_equality_to_none(self):
+        concern = WriteConcern()
+        none = None
```

Comment:
Aliasing `None` doesn't really make sense as it is a shared global constant. I would just use `None` directly in the assertions.

Reply:
It was to silence the pep8 warning (comparing with literal `None` should be done by using `is None`, however it's OK when it's a dynamic var that can have the `None` vaue). I've used a #noqa annotation instead.